art/launcher/sam/final/ic_launcher_foreground.svg

```diff
@@ -0,0 +1,61 @@
+           rdf:resource="http://purl.org/dc/dcmitype/StillImage" />
+        <dc:title></dc:title>
+      </cc:Work>
+    </rdf:RDF>
+  </metadata>
+  <sodipodi:namedview
+     pagecolor="#ffffff"
+     bordercolor="#666666"
+     borderopacity="1"
+     objecttolerance="10"
+     gridtolerance="10"
+     guidetolerance="10"
+     inkscape:pageopacity="0"
+     inkscape:pageshadow="2"
+     inkscape:window-width="1842"
+     inkscape:window-height="1051"
+     id="namedview4517"
+     showgrid="false"
+     inkscape:zoom="0.4609375"
+     inkscape:cx="256"
+     inkscape:cy="256"
+     inkscape:window-x="69"
+     inkscape:window-y="-9"
+     inkscape:window-maximized="1"
+     inkscape:current-layer="svg4569"
+     inkscape:pagecheckerboard="true" />
+  <defs
+     id="defs4563" />
+  <g
+     transform="translate(0,-161.53332)"
+     id="layer1"
+     style="fill:#ffffff;fill-opacity:1">
+    <path
+       style="fill:#ffffff;stroke-width:0.26458332;fill-opacity:1"
+       d="m 60.714159,256.96448 c -4.10854,-4.09497 -6.20706,-6.37507 -6.31354,-6.85985 -0.0941,-0.4283 0.002,-1.0325 0.23428,-1.47305 0.21657,-0.41081 2.99852,-3.33653 6.1821,-6.50161 5.7796,-5.746 5.79009,-5.75469 6.95145,-5.75469 1.16135,0 1.17185,0.009 6.95145,5.75469 3.18358,3.16508 5.96552,6.0908 6.1821,6.50161 0.23225,0.44055 0.32834,1.04475 0.23427,1.47305 -0.10647,0.48478 -2.20499,2.76488 -6.31354,6.85985 -5.6128,5.59426 -6.23323,6.13372 -7.05428,6.13372 -0.82105,0 -1.44148,-0.53946 -7.05429,-6.13372 z m -14.24467,-14.49672 c -0.29104,-0.12319 -3.23784,-2.94931 -6.54844,-6.28026 -5.482525,-5.51624 -6.019267,-6.13653 -6.019267,-6.95623 0,-0.8207 0.540189,-1.44194 6.133717,-7.05402 4.09497,-4.10855 6.37508,-6.20707 6.85986,-6.31354 0.4283,-0.0941 1.03249,0.002 1.47305,0.23428 0.4108,0.21657 3.33652,2.99851 6.5016,6.1821 5.746,5.7796 5.75469,5.79009 5.75469,6.95145 0,1.16135 -0.009,1.17185 -5.75469,6.95144 -3.16508,3.18359 -6.07619,5.95798 -6.46915,6.16532 -0.80055,0.42241 -1.16273,0.4448 -1.93137,0.11946 z m 40.48125,-0.23184 c -0.43656,-0.28045 -3.32383,-3.0596 -6.41615,-6.17589 -7.45896,-7.51681 -7.47227,-6.13092 0.13229,-13.77994 3.16508,-3.18359 6.09081,-5.96553 6.50161,-6.1821 0.44055,-0.23226 1.04475,-0.32835 1.47305,-0.23428 0.48478,0.10647 2.76488,2.20499 6.85985,6.31354 5.598231,5.61679 6.133721,6.23278 6.133721,7.05577 0,0.82341 -0.53422,1.43669 -6.151561,7.06186 -6.65021,6.65949 -7.01694,6.91483 -8.53281,5.94104 z M 60.816999,216.3331 c -3.18358,-3.16508 -5.96553,-6.0908 -6.1821,-6.50161 -0.23225,-0.44055 -0.32835,-1.04475 -0.23428,-1.47305 0.10648,-0.48477 2.205,-2.76488 6.31354,-6.85985 5.61281,-5.59426 6.23324,-6.13372 7.05429,-6.13372 0.82105,0 1.44148,0.53946 7.05428,6.13372 4.10855,4.09497 6.20707,6.37508 6.31354,6.85985 0.0941,0.4283 -0.002,1.0325 -0.23427,1.47305 -0.21658,0.41081 -2.99852,3.33653 -6.1821,6.50161 -5.7796,5.746 -5.7901,5.75469 -6.95145,5.75469 -1.16136,0 -1.17185,-0.009 -6.95145,-5.75469 z"
+       id="path4542" />
+  </g>
+</svg>
```

Comment:
This file would probably be better located at art/launcher/v26/

Reply:
Done.